src/components/__tests__/AllCountries.test.ts

```diff
@@ -1,3 +1,3 @@
-import AllCountries from '../../components/AllCountries'
+import AllCountries from '../AllCountries'
 
 console.log(AllCountries.getCountries())
```

Comment:
This could at least be a snapshot test since it does some refining logic before serving constants.

Reply:
> snapshot

@mcataford Sorry, I wouldn't know where to begin. I suppose you mean [render snapshot testing](https://jestjs.io/docs/snapshot-testing)?

Reply:
Exactly! It's crude, but if the point is to check the output value here and assert that it remains the same, it does the job. Alternatively, we could put something more thorough together but I think it goes beyond scope (I've got some ideas on this that I might be able to condense in a PR this week).